go.mod

```diff
@@ -29,6 +29,7 @@ require (
 )
 
 require (
+	github.com/alecthomas/kingpin/v2 v2.3.2 // indirect
```

Comment:
Please run command `go mod tidy`, it looks like this dependency id redundant.

Reply:
Done